LazyProxy/LazyProxyBuilder.cs

```diff
@@ -235,6 +235,7 @@ private static TypeBuilder AddMethods(this TypeBuilder typeBuilder, Type type, F
                     generator.Emit(OpCodes.Ldarg, i);
                 }
 
+                generator.Emit(OpCodes.Tail);
```

Comment:
Just wondering, why is the difference with [OpCodes.Tailcall](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.emit.opcodes.tailcall?view=net-8.0)? And also, does this optimization work for all calls or for the recursion calls only?

Reply:
Ah, `OpCodes.Tailcall` is correct vor .NET (fixed the code)
And `.Tail`  is for `Cecil` library used in DO

It works only before sequence `call`, `ret`
And this is not always recursive